src/js-tracker.js

```diff
@@ -49,8 +61,9 @@ function TrackerAsset() {
     this.active = false;
 
     this.session = 0;
-    this.actor = '{}';
+    this.actor = new TrackerEvent.TraceActor("undefined", "undefined", "undefined");
```

Comment:
Not a very good programming practice. Better allow null and check that is not null when the this.actor is required.

Reply:
fixed